src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Color {
+    YELLOW, BLUE, ORANGE, GREEN, BLACK, WHITE, PURPLE, RED, VIOLET, INDIGO
```

Comment:
After each comma that follows an enum constant, a line break is optional but when expanding the list it looks more readable